yarn.lock

```diff
@@ -783,6 +813,14 @@ babel-preset-jest@20.1.0-delta.1:
   dependencies:
     babel-plugin-jest-hoist "20.1.0-delta.1"
 
+babel-preset-minify@0.0.0:
+  version "0.0.0"
```

Comment:
It would still be nice to update it separately though :-) maybe it has improved in the meantime.